.github/workflows/conda-package.yml

```diff
@@ -291,7 +292,7 @@ jobs:
         env:
           DPCTL_VERBOSITY: error
         run: >-
-          conda activate dpctl_test && python -m pytest -v -s --pyargs ${{ env.MODULE_NAME }}
+          SET SYCL_QUEUE_THREAD_POOL_SIZE=6 && conda activate dpctl_test && python -m pytest -v -s --pyargs ${{ env.MODULE_NAME }}
```

Comment:
Instead of setting the variable in `run` section, add a line to `env` section on line 293:

```
   env:
     DPCTL_VERBOSITY: error
     SYCL_QUEUE_THREAD_POOL_SIZE: 6
  run:
      conda activate dpctl_test && python -m pytest -v -s --pyargs ${{ env.MODULE_NAME }}
```